docs/Unit-tests.md

```diff
@@ -0,0 +1,30 @@
+# Unit tests guide
+
+
+## Prerequisites
+
+You must have [Rake](http://rake.rubyforge.org/) installed on your testing environment before launch unit tests, you can install it using gem.
```

Comment:
before launch => before running
you can install it using gem => install it using `gem install rake`